lib/stimulus_reflex/html/fragment.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module StimulusReflex
+  module HTML
+    class Fragment < Part
+      def nokogiri_parser(html)
+        Nokogiri::HTML.fragment(html)
```

Comment:
So, now that we're pushing into the HTML5 dimension, can we also use `Nokogiri::HTML5::DocumentFragment` instead?

`Nokogiri::HTML4.fragment` calls `HTML4::DocumentFragment.parse` and returns a NodeSet.

`Nokogiri::HTML5::DocumentFragment` has a completely different mechanism. `parse` actually ends up calling `new`... but it also returns a NodeSet.

Reply:
I was able to make everything work `Nokogiri::HTML5.parse()`. So we don't need to differentiate between selector and page morphs anymore, which makes the whole code simpler again.